sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -267,7 +275,12 @@ class MessagesPaged(PageIterator):
     :param int max_messages: The maximum number of messages to retrieve from
         the queue.
     """
-    def __init__(self, command, results_per_page=None, continuation_token=None, max_messages=None):
+    def __init__(
```

Comment:
Anything that is `Paged` is not imported (look at __init__.py` to determine what is exported)
Big Callable